README.md

```diff
@@ -28,7 +28,7 @@ gem 'puppet-lint-absolute_classname-check', :require => false
 
 ### Relative class name inclusion
 
-Including a class by a relative name might lead to unexpected results.
+Including a class by a relative name might lead to unexpected results **in Puppet 3 only**. This plugin is **not** recommended for use with Puppet 4+.
```

Comment:
I wouldn't be so strong and give a bit more background: Proposal:

Puppet 3 uses relative includes when not prefixed with ```::```. This can lead to unexpected results. Since Puppet 4 all inclusions are absolute, whether they are prefixed with ```::``` or not. The check detects potential relative imports and helps you prepare your code base for Puppet 4 absolute imports.

Once your code base is has dropped support for Puppet 3 it is safe to stop using this plugin.

Reply:
I linked to the puppet doc explaining the issue with puppet 3 and strongly suggested dropping puppet 3.